wordpress-mobile/WordPress-iOS
# Feature/8232 tag settings

Closes #8232 

This is a pr to merge the Tags Settings feature written by @ctarda.   The work was reviewed in several installments and merged into this feature branch. See: #8236 #8244 #8248 #8278 #8288 #8305 
I would like a final test and review of the merged branch (for paranoia) of the feature branch before merging to develop. 

To test:
Be logged into wpcom. 
- Choose a site from My Sites and view its Settings > Tag Settings screen
- Confirm you can add new tags
- Confirm you can edit existing tags
- Confirm you can delete tags by swiping in the list or from the button in the detail view. 
- Confirm the list is updated in response to a newly created tag or a deleted tag.
- Confirm that tag descriptions are being updated.

Bonus, test with a self-hosted site (sans-Jetpack).  You'll need to edit the SiteSettingsViewController to view the Writing section in order to do this.  We've opted to leave the feature hidden from self-hosted for now but the support is there.

@elibud - fyi as this is relevant to your interests.
